src/NServiceBus.AcceptanceTesting/Support/TaskExtensions.cs

```diff
@@ -8,6 +8,7 @@
 
     static class TaskExtensions
     {
+        //this method will not timeout a task if the debugger is attached.
```

Comment:
Would it be worth making this into an XML doc comment so that it would show up in intellisense if someone tries to use it somewhere else?